test_scripts/Trainee/Updatting_DOP_value/003_Get_Vehicle_Data_Positive_with_empty_value_DOP.lua

```diff
@@ -0,0 +1,35 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0175-Updating-DOP-value-range-for-GPS-notification.md
+---------------------------------------------------------------------------------------------------
+-- In case:
+-- Mob app sends request GetVehicleData to SDL with gps parametr = true
+-- SDL resend this request to HMI with gps parameter = true
+-- HMI sends response without gps parameters: "pdop", "hdop", "vdop" to SDL
+-- SDL does:
+-- Resend notification to Mob App without gps parameters: "pdop", "hdop", "vdop"
+---------------------------------------------------------------------------------------------------
+-- [[ Required Shared libraries ]]
+local common = require('test_scripts/Trainee/Updatting_DOP_value/commonDOP')
+local runner = require('user_modules/script_runner')
+
+-- [[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+local value = nil
```

Comment:
In test response from HMI will be sent with empty gps data, so initial data must contain all params: pdop, vdop, hdop